src/application/F3DOptionsParser.cxx

```diff
@@ -378,13 +344,103 @@ void ConfigurationOptions::PrintHelp(cxxopts::Options& cxxOptions)
 //----------------------------------------------------------------------------
 void ConfigurationOptions::PrintVersion()
 {
-  f3d::engine::printVersion();
+  f3d::log::info(F3D::AppName + " " + F3D::AppVersion + "\n");
+  f3d::log::info(F3D::AppTitle);
+
+  auto libInfo = f3d::engine::getLibInfo();
+  std::vector<std::string> orderedInfo = { "Version", "Build date", "Build system", "Compiler",
+    "Raytracing module", "Exodus module", "OpenCASCADE module", "Assimp module", "Alembic module",
+    "VTK version", "License", "Authors" };
```

Comment:
missing python

Reply:
same here, python module is just a wrapper on top of libf3d. libf3d knows nothing about python.

Reply:
I think it is worth adding it here.